compilation break using clang

When compiling latest master of intel-vaapi-driver I got this error message:

```
gen9_avc_encoder.c:4981:9: error: initializer element is not a compile-time constant                                                                                                                  
        ((IS_SKL(i965->intel.device_info) || IS_BXT(i965->intel.device_info)) ? sizeof(gen9_avc_mbenc_curbe_data) : sizeof(gen95_avc_mbenc_curbe_data)),                                              
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~      
```

--- a/src/gen9_avc_encoder.c
+++ b/src/gen9_avc_encoder.c
@@ -4974,7 +4974,7 @@ gen9_avc_kernel_init_brc(VADriverContextP ctx,
     struct i965_kernel common_kernel;
     int i = 0;
 
-    static const int brc_curbe_size[NUM_GEN9_AVC_KERNEL_BRC] = {
+    const int brc_curbe_size[NUM_GEN9_AVC_KERNEL_BRC] = {
         (sizeof(gen9_avc_brc_init_reset_curbe_data)),
         (sizeof(gen9_avc_frame_brc_update_curbe_data)),
         (sizeof(gen9_avc_brc_init_reset_curbe_data)),

